pixi/src/ui/recommendations/RecommendationsView.js

```diff
@@ -65,11 +76,65 @@ export default class RecommendationsView {
 
       for (const tagId of value.tags) {
         const tag = this.tag.cloneNode(true);
+        recommendation.classList.add(tagId);
         tag.textContent = i18n.getText(`tags.${tagId}`);
         tagsBar.appendChild(tag);
+        tagIds.push(tagId);
       }
 
+      this.recommendationNodes.push(recommendation);
       this.container.appendChild(recommendation);
     }
+
+    for (const tagId of tagIds.values()) {
```

Comment:
Let's make sure we are only iterating through a unique set of `tagIds.values()`, otherwise we get this:
![image](https://user-images.githubusercontent.com/10456171/92607761-8dfc1680-f282-11ea-96df-7c6e031d3334.png)